src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+  background-image: url(/src/images/star.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(/src/images/star-active.svg);
```

Comment:
Similarly, the path to the active star image should also be relative to the CSS file. Change it to '../images/star-active.svg' instead of '/src/images/star-active.svg'.